test/api/SiteOrgTest.ts

```diff
@@ -65,6 +65,16 @@ async function assignUserToSite(userRole, site): Promise<any> {
   return testData.userService.siteApi.addUsersToSite(site.id, [userContext.id]);
 }
 
+/**
+ * @param userRole
+ * @param sites
+ */
```

Comment:
No need comments for method

Reply:
This is an ESLint auto generated JSDoc. Should we remove this rule ?

Reply:
> This is an ESLint auto generated JSDoc. Should we remove this rule ?

Yes, I think so.